exercises/matrix/matrix.py

```diff
@@ -1,3 +1,3 @@
 class Matrix(object):
-    def __init__(self):
+    def __init__(self, matrix):
```

Comment:
I think `matrix` is somewhat confusing name, since it just a string of numbers divided by newlines.
So, it would be more appropriate to name it as `matrix_string` or something else.

What do you think?

Reply:
True, made the required change.